docs/sdk/instructlab-sdk.md

```diff
@@ -0,0 +1,109 @@
+# InstructLab SDK
```

Comment:
probably obvious to you but consumers would be better served if we refer and publish the SDK as "Python SDK" (to contrast with whatever other bindings we could have in the future).